app/Http/Requests/SaveIndividualRolesRequest.php

```diff
@@ -34,6 +34,13 @@ public function prepareForValidation()
         request()->mergeIfMissing($fallbacks);
     }
 
+    public function attributes(): array
+    {
+        return [
+            'roles' => __('You must select what you would like to do on the website.'),
+        ];
+    }
```

Comment:
It looks like you copied the message here instead of adding the attribute value.

```suggestion
    public function attributes(): array
    {
        return [
            'roles' => __('roles'),
            'roles.*' => __('roles'),
        ];
    }
```